topo_processor/metadata/metadata_loaders/tests/metadata_loader_imagery_historic_test.py

```diff
@@ -308,3 +308,53 @@ def test_mission_metadata_not_added():
     metadata_loader_imagery_historic = MetadataLoaderImageryHistoric()
     title = metadata_loader_imagery_historic.get_title(metadata["survey"], metadata["alternate_survey_name"])
     assert title is None
+
+
+def test_test():
```

Comment:
You haven't been inspired for the name of your test! Maybe something like `test_provider_added`?

Reply:
:joy: oops!